spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasModule.scala

```diff
@@ -117,9 +117,15 @@ abstract class KerasModule[A <: Activity: ClassTag, B <: Activity: ClassTag, T:
   def doBuild(inputShape: Activity): AbstractModule[A, B, T]
 
 
-  override def forward(input: A): B = labor.forward(input)
+  override def forward(input: A): B = {
```

Comment:
revert this back?

Reply:
This is necessary. Otherwise for layers like `Reshape`, the reference of output has been changed when doing forward. If you don't change this, output will be an empty tensor, which will cause error when doing backward. Thanks to @qiuxin2012